aten/src/ATen/cpu/vec256/vec256_base.h

```diff
@@ -307,6 +307,15 @@ template <class T> Vec256<T> inline operator/(const Vec256<T> &a, const Vec256<T
   return c;
 }
 
+template <class T> Vec256<T> inline operator&&(
+    const Vec256<T> &a, const Vec256<T> &b) {
+  Vec256<T> c = Vec256<T>();
+  for (int i = 0; i != Vec256<T>::size(); i++) {
+    c[i] = a[i] && b[i];
```

Comment:
Question: are there vectorized bitwise and operators that you could use for avx2 etc?

Reply:
Good point! Let me check.

Reply:
@fmassa  @gchanan 
Let me try something as follows (will make generic and use loops):

```c++
Vec256<uint8_t> inline operator&&(const Vec256<uint8_t> &a, const Vec256<uint8_t> &b) {
  a = _mm256_or_si256(a, _mm256_srl_epi64(a, 4));
  a = _mm256_or_si256(a, _mm256_srl_epi64(a, 2));
  a = _mm256_or_si256(a, _mm256_srl_epi64(a, 1));
 
  // do the same for b

  return  mm256_and_si256(0x01010101..., mm256_and_si256(a, b));
}
```

Will rerun benchmark after this change. Comments are appreciated!